lib/orchparty/kubernetes_application.rb

```diff
@@ -322,25 +325,13 @@ def print(method)
     end
   end
 
-  def combine_charts(app_config)
```

Comment:
From my test this function makes from

```
[metoda-account-api-console
...
metoda-account]
```

this:

```[metoda-account]```


with my solution I have the same result. 
In the end we should build Charts...